Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -1,13 +1,59 @@
-variables:
-    SolutionBaseName: 'DFC.Composite.Shell'
-    BuildPlatform: 'any cpu'
-    BuildConfiguration: 'release'
-
 resources:
   repositories:
   - repository: self
   - repository: dfc-devops
     type: github
     name: SkillsFundingAgency/dfc-devops
-    ref: refs/tags/v1.6.5
-    endpoint: 'GitHub (ESFA)'
\ No newline at end of file
+    ref: refs/tags/v1.6.11
+    endpoint: 'GitHub (ESFA)'
+
+stages:
+- stage: Build
+  jobs:
+  - template: /AzureDevOpsTemplates/Build/dfc-arm-build.yml@dfc-devops
+    parameters:
+      ArmTemplateRoot: "$(System.DefaultWorkingDirectory)/Resources/ArmTemplates"
+
+  - job:
+    displayName: "Build DotNetCore app"
+    pool: 'NCS - CI and CD'
+    steps:
+    - task: CopyFiles@2
+      displayName: "Copy OpenAPI specification docs to artifacts"
+      inputs:
+        sourceFolder: "$(System.DefaultWorkingDirectory)/DFC.ServiceTaxonomy.ApiFunction/OpenAPIDocs"
+        contents: |
+          *.txt
+        targetFolder: '$(build.artifactstagingdirectory)\Artifact\DFC.ServiceTaxonomy.ApiFunction\OpenApiDocs'
+
+    - template: /AzureDevOpsTemplates/Build/dfc-dotnetcore-build.yml@dfc-devops
+      parameters:
+        SolutionBaseName: 'DFC.ServiceTaxonomy.ApiFunction'
+        BuildPlatform: 'any cpu'
+        BuildConfiguration: 'release'
+        PublishWebApp: 'false'
+        TestSuffix: 'Tests'
+
+- stage: DeployToDevServiceTaxonomy
+  displayName: Deploy to DEV_SERVICETAXONOMY
+  variables:
+  - group: dfc-stax-shared-dev
+  - group: dfc-shared-dev
```

Comment:
Not sure if this matters but it may make sense to order these by scope, ie variable groups shared most widely at the top

Reply:
re-ordered with most widely shared var groups at the top